code/modules/projectiles/projectile/magic.dm

```diff
@@ -130,9 +130,11 @@
 	. = ..()
 	var/atom/T = target.loc
 	if(isturf(target) && target.density)
-		CreateDoor(target)
+		if(!(istype(target, /turf/simulated/wall/indestructible)))
+			CreateDoor(target)
 	else if(isturf(T) && T.density)
-		CreateDoor(T)
+		if(!(istype(T, /turf/simulated/wall/indestructible)))
+			CreateDoor(T)
```

Comment:
I hate how this check looks redundant but is actually important.

Reply:
this is exactly what I thought